assignment+2/Assignment2PQ1.java

```diff
@@ -0,0 +1,105 @@
+public class Assignment2PQ1 {
+
+  // A custom linked list class
+  public static class MyLinkedList<G> {
+
+    public static class Node<G> {
```

Comment:
make this private and non static. if you want to have it globally available to the entire program, then declare the class outside of public static class MyLinkedList